README.rst

```diff
@@ -127,57 +125,57 @@ All the config files are working for both the 'post-synaptic cell only' and the
 
-Then create an Experiment instance:
+Then create an Experiment instance::
 
     exp = Experiment(data="output_50Hz_10ms.h5", c01duration=, c02duration=, period=)
 
 With data being the path to your output file, or a dictionary containing the output data, 
-c01duration, respectively c02duration, are the duration of the EPSP monitoring before, respectively after, the induction protocol,
+``c01duration``, respectively ``c02duration``, are the duration of the EPSP monitoring before, respectively after, the induction protocol,
 period is the period at which the synapses are stimulated.
 
-Note that the period argument can be replaced by c01period and c02period if the period used for the stimulation of the synapses is different after and before the induction protocol.
+Note that the period argument can be replaced by ``c01period`` and ``c02period`` if the period used for the stimulation of the synapses is different after and before the induction protocol.
 
-The change of EPSPs can then be computed by using the compute_epsp_ratio method, as follows:
+The change of EPSPs can then be computed by using the compute_epsp_ratio method, as follows::
 
     EPSP_ratio = exp.compute_epsp_ratio(n=60, method="amplitude)
 
-With n the number of sweeps to be considered for mean EPSP calculation, 
+With ``n`` the number of sweeps to be considered for mean EPSP calculation, 
 and method the method to use to compute EPSP ratio (can be "amplitude" or "slope").
 
 
 Sscx example
 ------------
 
-You can find an example cell package in example/sscx_sample_dir.
+You can find an example cell package in ``example/sscx_sample_dir``.
 Go to this folder, or to a cell package that has been designed to work with the sscx functions.
 
 Run the simulation using python
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Running the simulation should be as easy as:
+Running the simulation should be as easy as::
 
     sh run.sh config_path
 
-Where config_path is the path to a specific config file. You will find the available config files in the config folder.
+Where ``config_path`` is the path to a specific config file. You will find the available config files in the config folder.
 Note that the protocol used will depend on the contents of the config file.
 
-The output can be found under python_recordings.
+The output can be found under ``python_recordings``.
 
 Run the simulation using hoc
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can also run the simulation using hoc. In order to do that, you will have to first create the hoc files with the following line:
+You can also run the simulation using hoc. In order to do that, you will have to first create the hoc files with the following line::
 
     python create_hoc.py --config_path config_path
 
-Where config_path is the path to a specific config file. You will find the available config files in the config folder.
+Where ``config_pat``h is the path to a specific config file. You will find the available config files in the config folder.
```

Comment:
There is a type in config_path

Reply:
thanks for noticing, should be fixed now